devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,371 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using System.IO;
+using System.Numerics;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Mpu6886
+{
+    /// <summary>
+    /// Mpu6886 accelerometer and gyroscope
+    /// </summary>
+    [Interface("Mpu6886 accelerometer and gyroscope")]
+    public class Mpu6886AccelerometerGyroscope : IDisposable
+    {
+        private I2cDevice _i2c;
```

Comment:
Please add a `public const int DefaultI2cAddress = 0x68` with a proper description. This makes life easier for the developers.
Note, public constants goe first and require a /// comment.